src/aws/osml/tile_server/services/aws_services.py

```diff
@@ -1,18 +1,25 @@
 #  Copyright 2023-2024 Amazon.com, Inc. or its affiliates.
 
 import logging
+import sys
 import traceback
 from datetime import datetime, timezone
 from time import time
-from typing import Dict, Tuple
+from typing import Dict
 
 from boto3 import Session
 from boto3.resources.base import ServiceResource
 from botocore.credentials import RefreshableCredentials
+from botocore.exceptions import ClientError
 from botocore.session import get_session
 
 from aws.osml.tile_server.app_config import BotoConfig, ServerConfig
 
+from .database import ViewpointStatusTable
+from .queue import ViewpointRequestQueue
+
+logger = logging.getLogger("uvicorn")
```

Comment:
Why pulling the "uvicorn" logger directly here? Seems like it should be the module name. If this is just copy/paste move from existing code feel free to punt to a different PR but maybe open a ticket on our backlog to track that we need to look at how loggers are being setup in the TileServer.

Reply:
If we use a logger other than uvicorn and then run the app in uvicorn we don't get log messages from the other loggers.  A great fix? No, but not part of the scope of this PR.